web-common/src/components/chip/chip-types.ts

```diff
@@ -28,32 +28,3 @@ export const excludeChipColors = {
   outlineClass: "outline outline-1 outline-gray-200",
   outlineActiveClass: "!outline-gray-400",
 };
-
```

Comment:
note: ctrl+shift+f finds no usages of the three defs below in any code, so removing